airgun/views/task.py

```diff
@@ -20,12 +20,38 @@ class TasksView(BaseLoggedInView, SearchableViewMixin):
             'Action': Text('./a'),
         }
     )
+    focus = ActionsDropdown("//div[@class='dropdown btn-group']")
 
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
 
+    class RunningChart(View):
+        ROOT = ".//div[@id='running-tasks-card']"
+        name = Text("./h2")
+        total = PieChart("./div[@class='card-pf-body']")
+
+    class PausedChart(View):
+        ROOT = ".//div[@id='paused-tasks-card']"
+        name = Text("./h2")
+        total = PieChart("./div[@class='card-pf-body']")
+
+    class StoppedChart(View):
+        ROOT = ".//div[@id='stopped-tasks-card']"
+        name = Text("./h2")
+        table = SatTable(
```

Comment:
Wouldn't `Table` suffice here?

Andrii and Alex marked `SatTable` as deprecated and wanted to remove it completely, but did not finish that task. While I don't find it particularly high priority, let's not make our lives harder and don't use `SatTable` where `Table` is good enough.